src/Control/Session.php

```diff
@@ -354,6 +368,17 @@ public function start(HTTPRequest $request)
         $this->started = true;
     }
 
+    /**
+     * Determines what the value for the `secure` cookie attribute should be.
+     */
+    private function isCookieSecure(string $sameSite, bool $isHttps): bool
```

Comment:
This private function can be unit tested by using using reflection to change its visibility

Reply:
Test added